doc/source/whatsnew/v0.21.0.txt

```diff
@@ -147,7 +147,8 @@ Bug Fixes
 Conversion
 ^^^^^^^^^^
 
-
+- :func:`dtypes.common._get_dtype` now accepts ``Categorical``s and
```

Comment:
I don't quite understand. Above you say

> just needs a whatsnew note

Are you thinking of a different location for the note?